text/0000-process-stdio-redirection.md

```diff
@@ -0,0 +1,161 @@
+- Feature Name: process-stdio-redirection
+- Start Date: 2015-04-10
+- RFC PR:
+- Rust Issue:
+
+# Summary
+
+Update the standard library API with the ability to redirect stdio of child
+processes to any opened file handle or equivalent.
+
+# Motivation
+
+The current API in `std::process` allows to either pipe stdio between parent and
```

Comment:
Ah, I hadn't realized those changes had landed! I'll update the RFC to just focus on a high-level design